Gemfile.steep

```diff
@@ -1,4 +1,4 @@
 source "https://rubygems.org"
 
-gem 'steep', '~> 1.4.0.dev'
-gem "rbs", "~> 3.0.0.dev"
+gem 'steep', '~> 1.4.0.dev', require: false
+gem "rbs", path: "../rbs"
```

Comment:
I'm not sure what you have locally, but when I change this back to what you had before instead of using path, I can run `bin/setup` and `rake tests` leading to:
```sh
Finished in 153.540538s, 5.8030 runs/s, 34.0562 assertions/s.

891 runs, 5229 assertions, 0 failures, 0 errors, 1 skips
```

Reply:
For extra context:
```
gem environment
RubyGems Environment:
  - RUBYGEMS VERSION: 3.3.5
  - RUBY VERSION: 3.0.2 (2021-07-07 patchlevel 107) [x86_64-linux-gnu]
  - INSTALLATION DIRECTORY: /var/lib/gems/3.0.0
  - USER INSTALLATION DIRECTORY: /home/crashoverride/.local/share/gem/ruby/3.0.0
  - RUBY EXECUTABLE: /usr/bin/ruby3.0
  - GIT EXECUTABLE: /usr/bin/git
  - EXECUTABLE DIRECTORY: /usr/local/bin
  - SPEC CACHE DIRECTORY: /home/crashoverride/.local/share/gem/specs
  - SYSTEM CONFIGURATION DIRECTORY: /etc
  - RUBYGEMS PLATFORMS:
     - ruby
     - x86_64-linux
  - GEM PATHS:
     - /var/lib/gems/3.0.0
     - /home/crashoverride/.local/share/gem/ruby/3.0.0
     - /usr/local/lib/ruby/gems/3.0.0
     - /usr/lib/ruby/gems/3.0.0
     - /usr/lib/x86_64-linux-gnu/ruby/gems/3.0.0
     - /usr/share/rubygems-integration/3.0.0
     - /usr/share/rubygems-integration/all
     - /usr/lib/x86_64-linux-gnu/rubygems-integration/3.0.0
  - GEM CONFIGURATION:
     - :update_sources => true
     - :verbose => true
     - :backtrace => false
     - :bulk_threshold => 1000
  - REMOTE SOURCES:
     - https://rubygems.org/
  - SHELL PATH:
     - /home/crashoverride/.vscode-server/bin/74b1f979648cc44d385a2286793c226e611f59e7/bin/remote-cli
     - /usr/local/sbin
     - /usr/local/bin
     - /usr/sbin
     - /usr/bin
     - /sbin
     - /bin
     - /usr/games
     - /usr/local/games
     - /usr/lib/wsl/lib
     - /mnt/c/Program Files (x86)/NVIDIA Corporation/PhysX/Common
     - /mnt/c/Python311/Scripts/
     - /mnt/c/Python311/
     - /mnt/c/Windows/system32
     - /mnt/c/Windows
     - /mnt/c/Windows/System32/Wbem
     - /mnt/c/Windows/System32/WindowsPowerShell/v1.0/
     - /mnt/c/Windows/System32/OpenSSH/
     - /mnt/c/ProgramData/chocolatey/bin
     - /mnt/c/Program Files/Git/cmd
     - /mnt/c/ProgramData/nvm
     - /mnt/c/Program Files/nodejs
     - /mnt/c/Program Files/Docker/Docker/resources/bin
     - /mnt/c/ProgramData/DockerDesktop/version-bin
     - /mnt/c/Program Files/dotnet/
     - /mnt/c/Users/swens/AppData/Local/Microsoft/WindowsApps
     - /mnt/d/Microsoft VS Code/bin
     - /mnt/c/ProgramData/nvm
     - /mnt/c/Program Files/nodejs
     - /mnt/c/Users/swens/AppData/Local/GitHubDesktop/bin
     - /mnt/c/Users/swens/.dotnet/tools
     - /snap/bin
   ```

Reply:
@soutaro Here is my PR: https://github.com/soutaro/steep/pull/727/files